mkosi/backend.py

```diff
@@ -555,9 +557,16 @@ def write_grub_config(args: CommandLineArguments, root: Path) -> None:
     else:
 
         def jj(line: str) -> str:
-            if line.startswith("GRUB_CMDLINE_LINUX="):
+            if line.startswith("GRUB_CMDLINE_LINUX=") or line.startswith("#GRUB_CMDLINE_LINUX="): # GENTOO:
```

Comment:
```
line.startswith(("GRUB_CMDLINE_LINUX=", 
                               "#GRUB_CMDLINE_LINUX=")): # GENTOO:
```
?

Reply:
more or less, i think `black` was nagging if i broke the long lines.